### ISSUE TITLE
[feature] phpDocumentor v3 compatibility

### ISSUE THREAD

**Issue by nelson6e65** [nelson6e65/phpdoc-vuepress#18]
**Scopes**

- [x] Installation
- [x] Documentation
- [x] Template (twig, xml)
- [x] PHP sources
- [ ] Output HTML
- [x] Output Markdown
- [ ] Appearance
- [ ] Other

**Is your feature request related to a problem? Please describe.**
Provide compatibility with phpDocumentor 3.

**Describe the solution you'd like**
Drop compatibility with phpDocumentor 2, which is intended for old PHP versions (<7.2). And provide configuration and documentation for new phpDocumentor 3 and recent PHP versions.

**Comment by eshimischi** [comment]
Any progress here?

**Comment by nelson6e65** [comment]
Sadly, there's no recent work on this project. 🥲

I was starting a VitePress templating with phpDocumentor3, but I got no spare time to work more on it neither (not published, unnestable).

I'll try this next weekends to check if is more practical to continue working on **VuePress** or if I should continue with my **VitePress** version instead.

**Comment by eshimischi** [comment]
Well, first of all, your project is very useful! And keeping idea in mind that we are now php8, we have to make a step forward with new tools and stuff. If you are able to share your VitePress version too, perhaps i'll find my time to help or whatsoever move it forward as well as phpDocumentor 3 version of VuePress. Thanks in advanced!

**Comment by nelson6e65** [comment]
Yes, of course! When I tried to use phpdoc v2 it failed on PHP > 7.2. That's why I tried v3, but it was still on beta and had no documentation, even after stable 3.0 release it was difficult to find out any information about new templating and configuration.

About VitePress, that time I tried, but it was not compatible with the way phpdoc worked. I think that, with the coming VuePress 2, there's an opportunity to use it easier that VitePress.

I'll poke you, @eshimischi , when I test these ideas this weekend. Thanks for your interest. 😸

**Comment by eshimischi** [comment]
Awesome, let's do it. I'll find my time to support it. Really interested in the project. Looking forward!

**Comment by eshimischi** [comment]
Any progress here?

**Comment by nelson6e65** [comment]
No, sorry. I'm struggling with my free time x.x

**Comment by eshimischi** [comment]
> No, sorry. I'm struggling with my free time x.x

it's ok i was just curious about progress, no pressure. take your time!
